src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs

```diff
@@ -117,5 +120,45 @@ public void get_website_address_manually_entered_returns_no_value_when_not_prese
 
             Assert.IsNull(actualResult);
         }
+
+        [TestCase(1, "www.UkrlpWebsite.co.uk", "www.ApplyWebsite.co.uk")]
+        [TestCase(2, "www.UkrlpWebsite.co.uk", null)]
+        [TestCase(3, null, "www.ApplyWebsite.co.uk")]
+        [TestCase(4, null, null)]
+        public void get_organisation_website_address_returns_expected_value_when_present(int testCase, string ukrlpWebsite, string applyWebsite)
+        {
+            _qnaApiClient
+               .Setup(x => x.GetAnswerValue(_applicationId,
+                                           RoatpWorkflowSequenceIds.Preamble,
+                                           RoatpWorkflowSectionIds.Preamble,
+                                           RoatpWorkflowPageIds.Preamble,
+                                           RoatpPreambleQuestionIdConstants.UkrlpWebsite)).ReturnsAsync(ukrlpWebsite);
+
+            _qnaApiClient
+               .Setup(x => x.GetAnswerValue(_applicationId,
+                                           RoatpWorkflowSequenceIds.YourOrganisation,
+                                           RoatpWorkflowSectionIds.YourOrganisation.WhatYouWillNeed,
+                                           RoatpWorkflowPageIds.WebsiteManuallyEntered,
+                                           RoatpYourOrganisationQuestionIdConstants.WebsiteManuallyEntered)).ReturnsAsync(applyWebsite);    
+
+
+            var actualResult = _controller.GetOrganisationWebsiteAddress(_applicationId).Result;
+
+            switch (testCase)
```

Comment:
This definitely shouldn't be here.  If we are checking the case number we should have 4 different tests.

Reply:
Obviously reuse all of the setup above tho, no need to copy it all to each test.

Reply:
Tests are refactored.